github/ci/prow/files/labels.yaml

```diff
@@ -148,6 +148,10 @@ default:
     - name: topic/virtualization
       color: c5def5
       target: both
+    - name: build-officer
+      color: 8452c9
+      target: issues
```

Comment:
nit: maybe `both`? As the build-officer might also create PRs directly?

Reply:
In theory yes, but practically, build officer open bug. Teams fixes. So maybe it's the same person, but with different "hat"